dnguyen09/solvd-laba-test-autiomation
# Mobile test

Add 4 more test cases:

Test case 1: Test Add City funtion
Description: Test the world clock feature in the clock app to add different city's time
Step:
	1. Locate and tap on the lock app icon on the clock app home screen - the clock app should open, display current time
	2. Tap on the "+" sign button to add city - the clock app should present an input field to enter city name
	3. in the input field, type the name of the city - the city should be available for adding
	4. Verify that the city's time is added
	
Test Restult: the clock app successfully added new city and its time

Test case 2: Test Time and Date Display
Description: Test the clock app display the current time and date accurately
Step:
	1. Locate and tap on the lock app icon to open the app 
	2. Verify that the time and date displayed on the clock match the current time and date on the device

Test result:  the clock app displays the current time and date accurately

Test case 3: Test Timer Functionality
Description: test the timer feater in the clock app
Step:
	1. Locate and tap on the lock app icon to open the app
	2. tap on timer icon - timer page should open
	3. Set timer to 3 second - the timer is set to 3 second
	4. Tap on start button and wait for 3 second - the timer start couting down
	5. Verify that when the timer reachs zero, an alarm should should ring

Test result: the timer accurately counts down from the set duration, and an alarm rings when the timer reaches zero	

Test case 4: Test Alarm functionality
Description: Verify the alarm function displays time accurately after configuraion
Step:
	1. Locate and tap on the lock app icon to open the app
	2. Tap on alarm icon - alarm feature is accessible
	3. Set the alarm time - the alarm is set to specific hour, minute, and AM/PM
	4. Tap on "ok" button
	5. Verify the display alarm time - the display time should match the set alarm time
	
Test Result: the alarm functionality allows users to set an alarm time, and the time is displayed correctly after configuration

Test case 5: Test Change Hour Setting
Description: test case to verify the clock app's ability to swtich between 12-hour and 24-hour format
Step:
	1. Locate and tap on the lock app icon to open the app
	2. Tap on the three dot option - the option should be displayed
	3. Tap on settings - settings page should  open
	4. From settings page, tap on change Date and Time - Date and Time page should open
	5. At Date and Time page, toggle the switch from 12-hour to 24-hour format
	6. Verify that the time format change accordingly
	
Test Result: the clock app allows user to toggle from 12-hour to 24-hour and displays the time format change accordingly
